docs/tutorials/see-all-tutorials.md

```diff
@@ -23,9 +23,14 @@
 * [Encrypted Key-value Database Using Homomorphic Encryption](https://www.zama.ai/post/encrypted-key-value-database-using-homomorphic-encryption) - March 2023
 
 #### Video tutorials
-* [Compute an XOR distance in FHE using Concrete](https://www.zama.ai/post/video-tutorial-compute-an-xor-distance-in-fhe-using-concrete) - May 2024
-* [Speed up neural networks with approximate rounding using Concrete](https://www.zama.ai/post/video-tutorial-speed-up-neural-networks-with-approximate-rounding-using-concrete) - May 2024
-* [Compile composable functions with Concrete](https://www.zama.ai/post/video-tutorial-compile-composable-functions-with-concrete) - February 2024
-* [How to use dynamic table look-ups using Concrete](https://www.zama.ai/post/video-tutorial-how-to-use-dynamic-table-look-ups-using-concrete) - October 2023
-* [Dive into Concrete - Zama's Fully Homomorphic Encryption Compiler](https://www.zama.ai/post/video-tutorial-dive-into-concrete-zamas-fully-homomorphic-encryption-compiler) - October 2023
-* [How To Get Started With Concrete - Zama's Fully Homomorphic Encryption Compiler](https://www.zama.ai/post/how-to-started-with-concrete-zama-fully-homomorphic-encryption-compiler)  - July 2023
+
```

Comment:
Is that removing of XOR and Speedup neural networks are intentional? (cc @yuxizama @bcm-at-zama )

Reply:
Hi Quentin, no I don't think it's intentional. 
I can't find where these changes come from. Could you add them back, please?

Reply:
yes if you can re-add them, it's great, thx @BourgerieQuentin